detection_rules/rule.py

```diff
@@ -245,6 +281,19 @@ class EQLRuleData(QueryRuleData):
     type: Literal["eql"]
     language: Literal["eql"]
 
+    def get_query_schema(self, ast: Union[eql.ast.Expression, kql.ast.Expression], beats_version: str,
```

Comment:
Should this be on the query validator now instead, not on this class directly?

Reply:
7a90cc3

Reply:
I think that's the wrong commit hash, and I don't see any changes yet

Reply:
sorry, it was unused residual 83803c4